src/me.rs

```diff
@@ -511,15 +511,14 @@ impl MotionEstimation for DiamondSearch {
     blk_w: usize, blk_h: usize,
     best_mv: &mut MotionVector, lowest_cost: &mut u64
   ) {
-    let bo_adj_h = BlockOffset { x: bo_adj.x >> 1, y: bo_adj.y >> 1 };
```

Comment:
RIght, apparently the master is doing it wrong. Because bo_adj_h is already half-resolution block offset then it divide it by 2 again, making it to 1/4 resolution bo!